apps/volunteer/sign_up.py

```diff
@@ -23,28 +26,13 @@
 from ..common import create_current_user, feature_flag
 
 
-def generate_day_options(start, stop):
-    days = period(parse(start), parse(stop)).range("days", 1)
-    return list([(d.strftime("%Y-%m-%d"), d.strftime("%A %-d %B")) for d in days])
-
-
-ARRIVAL_CHOICES = [
-    ("2018-08-29", "Wednesday 29 August or earlier")
-] + generate_day_options("2018-08-30", "2018-09-02")
-DEPARTURE_CHOICES = generate_day_options("2018-08-31", "2018-09-03") + [
-    ("2018-09-04", "Tuesday 4 September or later")
-]
-
-
 class VolunteerSignUpForm(Form):
     nickname = StringField("Name", [Required()])
     volunteer_email = StringField("Email", [Email(), Required()])
     over_18 = BooleanField("I'm at least 18 years old")
     volunteer_phone = StringField("Phone", [Required()])
```

Comment:
@russss Out of interest, do you know if we could/should switch this over to use a [TelField](https://github.com/emfcamp/Website/blob/e37dcb141ab97c93d9d68e8d3887dbee8cb763fc/apps/common/forms.py#L56)?  (not directly related to the changes in this PR)

Reply:
I'd say that this is out of scope for a bug fix to automatically calculate dates rather than having them manually hard coded.

Reply:
Yeah, not in scope. I don't know off hand what TelField does so I can't say if it's appropriate.